.github/workflows/production-tag.yml

```diff
@@ -0,0 +1,128 @@
+name: Create Production Tag
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ['Continuous Integration']
+    types: [completed]
+    branches: [main]
+
+concurrency:
+  group: production-tag
+  cancel-in-progress: true
+
+env:
+  SLACK_CHANNEL: C06DSBT7CBW #status-next-build
+  DSVA_SCHEDULE_ENABLED: true
+
+jobs:
+  create-production-tag:
+    name: Create Production Tag
+    runs-on: ubuntu-latest
+    # Run the workflow unless it was triggered by CI and that failed
+    if: ${{ !(github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'failure') }}
```

Comment:
We cannot prevent a failed but completed CI run from triggering the workflow, so we have this here to prevent the tagging from running if it was a failure.